media/css/firefox/developer/whatsnew-mdnplus.scss

```
@@ -91,90 +82,48 @@ $image-path: '/media/protocol/img';
     }
 
     @keyframes mandala-color-change {
-        0% {
-            fill: var(--mandala-primary);
-        }
-
-        10% {
-            fill: var(--mandala-primary);
-        }
-
-        15% {
-            fill: var(--mandala-accent-1);
-        }
-
-        20% {
-            fill: var(--mandala-primary);
-        }
-
-        25% {
-            fill: var(--mandala-primary);
-        }
-
-        30% {
-            fill: var(--mandala-accent-2);
-        }
-
-        35% {
-            fill: var(--mandala-primary);
-        }
-
-        40% {
-            fill: var(--mandala-primary);
-        }
-
-        50% {
-            fill: var(--mandala-accent-3);
-        }
-
-        55% {
-            fill: var(--mandala-primary);
-        }
-
-        60% {
-            fill: var(--mandala-primary);
-        }
-
-        65% {
-            fill: var(--mandala-accent-4);
-        }
-
-        70% {
-            fill: var(--mandala-primary);
-        }
-
-        100% {
-            fill: var(--mandala-primary);
-        }
-    }
-
-    &.animated {
-        .mandala-rotate > svg {
+        0% { fill: var(--mandala-primary); }
+        10% { fill: var(--mandala-primary); }
+        15% { fill: var(--mandala-accent-1); }
+        20% { fill: var(--mandala-primary); }
+        25% { fill: var(--mandala-primary); }
+        30% { fill: var(--mandala-accent-2); }
+        35% { fill: var(--mandala-primary); }
+        40% { fill: var(--mandala-primary); }
+        50% { fill: var(--mandala-accent-3); }
+        55% { fill: var(--mandala-primary); }
+        60% { fill: var(--mandala-primary); }
+        65% { fill: var(--mandala-accent-4); }
+        70% { fill: var(--mandala-primary); }
+        100% { fill: var(--mandala-primary); }
+    }
+
+    @media not (prefers-reduced-motion: reduce) {
+        body:hover & {
+            // motion
```

BTW… If you're thinking easter egg, say on pointer events, it can animate e.g. on CTA hover:

```suggestion
    @media not (prefers-reduced-motion: reduce) {
        .c-mdn-header:has(.c-mdn-header-cta a:hover) & {
            // motion
```

(or any other crazy selector crossing DOM levels if needed, as it only targets FXDE so a pretty current beta can be expected, so `:has()` support since Fx121 is good enough here?)

---

_(You can check out https://github.com/janbrasna/bedrock/commit/6b40a0a812d30da231975ec5e2c29143a4201b07 if you fancy trying out the change above ^^)_